### ISSUE TITLE
 asdf list-all dotnet shows duplicates

### ISSUE THREAD

**Issue by marcinjahn** [hensou/asdf-dotnet#22]
**Describe the bug**

Running ` asdf list-all dotnet` shows:

```
1.1.14
1.1.14
2.1.202
2.1.818
2.2.207
3.0.103
3.1.426
5.0.408
6.0.417
7.0.404
8.0.100
```

The `1.1.14` version is listed twice.

**Steps to reproduce**

Run `asdf list-all dotnet`

**Expected behavior**

Each unique version should be listed just once.

**Comment by hensou** [comment]
Thanks for your bug report, this will get fixed when https://github.com/hensou/asdf-dotnet/pull/31 gets merged

**Comment by hensou** [comment]
Fixed in #31
